package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "devDependencies": {
+    "@symfony/webpack-encore": "^0.7.1",
+    "node-sass": "^4.5.3",
+    "sass-loader": "^6.0.5"
+  },
+  "dependencies": {
+    "bloodhound-js": "^1.2.1",
+    "bootstrap-sass": "^3.3.7",
+    "bootstrap-tagsinput": "^0.7.1",
+    "bootswatch": "^3.3.7",
+    "eonasdan-bootstrap-datetimepicker": "^4.17.47",
+    "font-awesome": "^4.7.0",
+    "highlight.js": "^9.12.0",
+    "imports-loader": "^0.7.1",
+    "jquery": "^3.2.1",
+    "lato-font": "^3.0.0",
+    "moment": "^2.18.1",
+    "typeahead.js": "^0.11.1"
+  }
```

Comment:
These should all go into `devDependencies`. It doesn't really matter, but the idea is that you really only need all of this stuff while your "developing". You won't need these files on production, because you will have already built your final files and are shipping those statically.